java/flight/flight-core/pom.xml

```diff
@@ -299,32 +288,21 @@
         </executions>
       </plugin>
     </plugins>
-    <extensions>
-      <extension>
-        <groupId>kr.motd.maven</groupId>
-        <artifactId>os-maven-plugin</artifactId>
-        <version>1.7.1</version>
-      </extension>
-    </extensions>
```

Comment:
We used this since some gRPC artifacts are platform-dependent. Are the properties this plugin provides now provided by something else?

Reply:
It's already declared by the top-level parent [pom.xml ](https://github.com/apache/arrow/blob/4b2cf22f5aea16f4ad0db447624b6e713b43d62c/java/pom.xml#L847)